Automattic/wp-calypso
# Odyssey Widget: use container query with media query as fallback

Related to #76380

## Proposed Changes

- Replaces Media Query with [Container Query](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries) , which has ~86% support, which is not-bad support rate for browsers.
- Fallback to Media Query
- Uses `<=599px` as a narrow breakpoint

A note here, named container doesn't seem to work on Firefox - we only used the unnamed container solution.

## Testing Instructions

* Adjust the width of the screen on Chrome, Firefox and Safari
* Ensure the widget switches to mobile layout when the container is narrower than `600px`
* Ensure it works well on Android Chrome (40% of market share)

<img width="1358" alt="image" src="https://user-images.githubusercontent.com/1425433/235570238-74d13fbe-6c92-4f53-807a-a3df0c0e69d0.png">



## Pre-merge Checklist
